Xamarin.Forms.Maps.UWP/MapRenderer.cs

```diff
@@ -176,20 +177,22 @@ void LoadPin(Pin pin)
 						LoadUserPosition(userPosition.Coordinate, moveToLocation);
 				}
 
+				if (Control == null || Element == null) return;
```

Comment:
@rmarinho I'm happy to keep the null checks consistent. The reason I included a check on both lines 180 and 167, is because if the user exits the Map page while the geolocater is still operating, the check on line 167 would not be enough.